experimental/dds/tree2/src/feature-libraries/sequence-change-family/sequenceChangeset.ts

```diff
@@ -3,11 +3,14 @@
  * Licensed under the MIT License.
  */
 
+import { Type } from "@sinclair/typebox";
 import { ICodecFamily, makeCodecFamily, makeValueCodec } from "../../codec";
 import { Transposed as T } from "./changeset";
 
 export type SequenceChangeset = T.LocalChangeset;
 
 export const sequenceChangeCodecs: ICodecFamily<SequenceChangeset> = makeCodecFamily([
-	[0, makeValueCodec<SequenceChangeset>()],
+	// This sequence field encoding is slotted for deletion. No effort has been made to
```

Comment:
Maybe mark this as a `TODO` for good measure

Reply:
```suggestion
	// TODO: This sequence field encoding is slotted for deletion. No effort has been made to
```